public/app/features/alerting/unified/NotificationPolicies.tsx

```diff
@@ -191,8 +191,11 @@ const AmRoutes = () => {
   if (!selectedAlertmanager) {
     return null;
   }
-
-  const numberOfMuteTimings = result?.alertmanager_config.mute_time_intervals?.length ?? 0;
+  const time_intervals = [
+    ...(result?.alertmanager_config?.mute_time_intervals ?? []),
+    ...(result?.alertmanager_config?.time_intervals ?? []),
+  ];
+  const numberOfMuteTimings = time_intervals?.length ?? 0;
```

Comment:
nit: I don't think `time_intervals` can be undefined when we spread in to a new Array

Reply:
💯